src/components/addSourceWizard/__tests__/__snapshots__/addSourceWizardStepOne.test.js.snap

```diff
@@ -86,6 +86,27 @@ exports[`AddSourceWizardStepOne Component should render a basic component: basic
             data-ouia-component-id="OUIA-Generated-Radio-3"
             data-ouia-component-type="PF4/Radio"
             data-ouia-safe="true"
+            id="sourceType-acs"
+            name="sourceType"
+            type="radio"
+            value="acs"
+          />
+          <label
+            class="pf-c-radio__label"
+            for="sourceType-acs"
+          >
+            t(form-dialog.label, {"context":"acs"})
```

Comment:
Same here - I don't know why, but "ACS" is all lowercase.

![image](https://github.com/quipucords/quipucords-ui/assets/12373754/51bdbf62-7b47-48c5-9eab-ed2324637eba)

Reply:
This appears to be resolved too!